### ISSUE TITLE
Cannot attach more storage unit to the heart, unit appear active all the time

### ISSUE THREAD

**Issue by SenLongYu** [blushiemagic/MagicStorage#141]
![20211124172021_1](https://user-images.githubusercontent.com/94999339/143320365-f189fb63-dec9-4230-82d0-23979b383840.jpg)
As seen in the pic, the left chain of the storage system cannot be added onto it. I try to add more units to the system and the capacity does not increase, the already attached units can be upgraded.

Also, I placed a unit standalone on the right, and it appear active (yellow dot top right, green bot right), the unit cannot be upgraded and when I right-click on it, it says "active 0/40).

Most recent version of the mod and stable tmodloader on 1.3.5

**Comment by ExterminatorX99** [comment]
Does this happen with only Magic Storage enabled?

**Comment by SenLongYu** [comment]
> Does this happen with only Magic Storage enabled?

No, I'm playing a small mod pack (calamity/thorium). The mod seems completely broken for me, as seen below that the crafting interface kept saying it is not connected to a crafting heart.
![20211124175143_1](https://user-images.githubusercontent.com/94999339/143322754-c35643ca-fba2-460d-87a5-a0f56f6a03d0.jpg)
.

**Comment by ExterminatorX99** [comment]
What I meant is, does this issue happen when you disable all mods except `MSE`?
Does breaking and placing the `Storage Heart` help?
Could you send your mod list?

**Comment by SenLongYu** [comment]
> What I meant is, does this issue happen when you disable all mods except `MSE`? Does breaking and placing the `Storage Heart` help? Please send your mod list.

After a closer inspection of the info provided by [What Mod Is This From] mod, I noticed that I am partially crafting elements from a mod called [Magic Storage - Extra], when I crafting elements for the mod [Magic Storage], everything seems to work as expected.

**Comment by ExterminatorX99** [comment]
Ahh. Yeah. They're 2 completely separate mods. You don't need one to use the other.
MSE is not an addon

**Comment by absoluteAquarian** [comment]
The following message has been copy/pasted to all issues older than 3 months.

In order to clean up how the project looks, I'll be closing all old issues.  If you believe that this issue is still relevant, please make a new issue that references this issue in its message.
